src/Unleash/Internal/Variant.cs

```diff
@@ -2,21 +2,18 @@
 
 namespace Unleash.Internal
 {
-    public class Variant
+    public class Variant: Yggdrasil.Variant
```

Comment:
What do we need to keep this one around for? I can't remember

Reply:
Well, `Variant` is a public class so I assume we still want it available. If we don't I'm more than happy to drop this entirely and just rely on `Yggdrasil.Variant` internally. WDYT?

Reply:
>  What do we need to keep this one around for? I can't remember

In theory, it stops people having to run around like crazy people updating their namespaces. Just makes this upgrade a slightly less bitter pill to swallow

Reply:
Ok if you're happy keeping it around then sure. It could be part of the migration article mentioned above though, since they're already having to do a bunch of work